107/src/main/java/org/ehcache/jsr107/internal/Jsr107CacheConfigurationParser.java

```diff
@@ -52,7 +54,7 @@ public URI getNamespace() {
   }
 
   @Override
-  public ServiceConfiguration<Jsr107Service> parseServiceConfiguration(Element fragment) {
+  public ServiceConfiguration<Jsr107Service> unmarshalServiceConfiguration(Element fragment) {
```

Comment:
You can't change the name of an SPI method without good reason they're considered semi-stable. Not as stable as public API, but we don't break things without good reason.

Reply:
This is why I recommended parse and unparse - it fits with the current naming scheme.

Reply:
I was not sure whether to go for marshal/unmarshal or parse/unparse. Now it is clear. I will make the change. Methods names will start with parse/unparse